dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -538,6 +538,18 @@ def _preprocess_for_calculate_psi(
             and isinstance(other_histogram["bin_counts"], np.ndarray)
             and isinstance(other_histogram["bin_edges"], np.ndarray)
         ):
+
+            if (
+                len(self_histogram["bin_counts"]) <= 1
+                and len(other_histogram["bin_counts"]) <= 1
+            ):
+                warnings.warn(
+                    """Data were essentially the same and less
+                    than or equal to one bin. PSI cannot be performed.""",
+                    RuntimeWarning,
+                )
+                return new_self_histogram, new_other_histogram
```

Comment:
Not certain of `return new_self_histogram, new_other_histogram`. 

It does trigger the `return None` in the `_calculate_psi` function, which is a good thing because if the logic on L543-544 return `True` I don't want PSI to even be attempted to be calculated because we have all the values in one bin_edge. **This is the bug.**

Another route, instead of returning `new_self_hist` and `new_other_hist` is to return `self_hist` and `other_hist`, but that is basically kicking us out of the `preprocessing` with two return values that is essentially saying "this is processed data and usable in `_calculate_psi`. Whereas, returning `new_self_hist` and `new_other_hist` is returning something and when `_calcl_psi` sees it it knows to kick out and not calc PSI.... 

open to thoughts here or where / better way to implement this logic

Reply:
need to return `0`